client/client.go

```diff
@@ -119,11 +119,9 @@ func (c *Client) Get(stackName string) (*StackInfo, error) {
 
 // GetTemplate retrieves a stack's underlying template
 func (c *Client) GetTemplate(stackName string) (string, error) {
-	input := &cf.GetTemplateInput{
-		StackName:     aws.String(stackName),
-		TemplateStage: aws.String(cf.TemplateStageProcessed),
```

Comment:
We didn't seem to be using this processed version, and the original is the one we want to diff... I think.